Refresher/Patching/EbootPatcher.cs

```diff
@@ -96,17 +120,102 @@ private static List<PatchTargetInfo> FindPatchableElements(Stream file)
         List<PatchTargetInfo> foundItems = new();
         FilterValidUrls(reader, possibleUrls, foundItems);
         FindDigestAroundCookie(reader, cookiePositions, foundItems);
+        FindLbpkDomains(reader, lbpkPositions, foundItems);
+        FindLbpkPorts(reader, lbpkPortPositions, foundItems);
 
         long end = Stopwatch.GetTimestamp();
         Console.WriteLine($"Detecting patchables took {(double)(end - start) / (double)Stopwatch.Frequency} seconds!");
         return foundItems;
     }
 
+    private static int SwapEndianness(int num)
+    {
+        return (num & 0x000000FF) << 24 |
+               (num & 0x0000FF00) << 8 |
+               (num & 0x00FF0000) >> 8 |
+               (int)((num & 0xFF000000) >> 24);
+    }
+
+    private static void FindLbpkPorts(BinaryReader reader, List<long> foundLbpkPortPositions, List<PatchTargetInfo> foundItems)
+    {
+        foreach (long foundPosition in foundLbpkPortPositions)
+        {
+            reader.BaseStream.Position = foundPosition;
+            int port1 = SwapEndianness(reader.ReadInt32());
+            int port2 = SwapEndianness(reader.ReadInt32());
```

Comment:
These are technically unsigned 16 bit integers stored into 4 byte regions, but since every unsigned 16 bit integer can fit into a signed 32-bit integer, it doesnt matter in this case